lib/util/fs/index.ts

```diff
@@ -160,6 +174,18 @@ export async function localPathIsFile(pathName: string): Promise<boolean> {
   }
 }
 
+export async function localPathIsSymbolicLink(
+  pathName: string
+): Promise<boolean> {
+  const path = ensureLocalPath(pathName);
+  try {
+    const s = await fs.lstat(path);
+    return s?.isSymbolicLink() ?? false;
```

Comment:
```suggestion
    return s.isSymbolicLink();
```

Sorry, it seems it never returns null. My fault.